level2/instance.tf

```diff
@@ -31,20 +30,35 @@ resource "aws_security_group" "public" {
 }
 
 
-resource "aws_instance" "public" {
+#resource "aws_instance" "public" {
```

Comment:
could you please remove commented codes and also reformat your code using `terraform fmt `